modules/templates/htmlrenderer.go

```diff
@@ -96,6 +99,16 @@ func HTMLRenderer() *HTMLRender {
 	return htmlRender
 }
 
+func ReloadHTMLTemplates(test bool) error {
+	if err := htmlRender.CompileTemplates(test); err != nil {
+		if !test {
+			log.Error("Template error: %v\n%s", err, log.Stack(2))
+		}
+		return err
+	}
+	return nil
```

Comment:
I think the code be as simplified to `return htmlRender.CompileTemplates(test)`, let caller decide whether print the error?